exercises/concept/calculator-conundrum/.docs/hints.md

```diff
@@ -0,0 +1,15 @@
+# Hints
+
+### 1. Implementing operations
```

Comment:
This needs to be a heading 2 (so `##`) and the text needs to exactly match the text in the instructions. Same throughout this file.